cds_ils/migrator/eitems/api.py

```diff
@@ -233,33 +236,44 @@ def migrate_ebl_links():
     search = get_documents_with_ebl_eitems()
     click.echo("Found {} documents with ebl links.".format(search.count()))
 
+    url_template = \
+        "http://ebookcentral.proquest.com/lib/cern/detail.action?docID={}"
```

Comment:
Can we use HTTPS? :)
Can you move this close to the EBL part?

Reply:
I'm not sure what you mean by
> move this close to the EBL part